share/spack/templates/modules/modulefile.tcl

```diff
@@ -63,7 +63,7 @@ unsetenv {{ cmd.name }}
 {# We are using the usual separator #}
 {% if command_name == 'PrependPath' %}
 prepend-path {{ cmd.name }} "{{ cmd.value }}"
-{% elif command_name == 'AppendPath' %}
+{% elif command_name in ('AppendPath', 'AppendFlagsEnv') %}
```

Comment:
This is more of a stylistic question for @alalazo, but why do we care if `cmd.separator` is "standard" or not?  Could we simplify the code and have only one case here?  Or did people want their tcl modules to look nice and not have a `--delim` for `append-path` unless it was really needed?

If we do want to check `separator`, then I *think* `AppendFlagsEnv` only needs to be considered in the "nonstandard" part of this PR, as I think it can only ever have space as its separator.  Is that right @becker33?

Reply:
Yes, it can only have a space as its separator.

Reply:
We have the two options because at the time there was a request for support of an old `environment-modules` version, dating 2002, which doesn't have the `--delim` option see https://github.com/spack/spack/pull/1358#issuecomment-242084389 Since we are 4 years later I would just remove the "standard" branch and support only `--delim`.

Reply:
@RikkiButler20 can you go ahead and implement @alalazo's suggestion here?